facturacion_mexico/facturacion_fiscal/tests/test_derive_pac_reconciliation.py

```
@@ -0,0 +1,211 @@
+			"docstatus": 1,
+		}
+	)
+	ffm.flags.ignore_validate = True
+	ffm.flags.ignore_mandatory = True
+	ffm.flags.ignore_links = True
+	ffm.db_insert()
+	frappe.db.commit()
+	return ffm.name
+
+
+def _set_value_factory():
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_set_value_factory`

(ANN202)